requirements/constraints.txt

```diff
@@ -0,0 +1,19 @@
+# Version constraints for pip installation.
```

Comment:
Nice!  Is a `constraints.txt` file something we've used before as well?

Reply:
Yea,  we started using it in edx-platform a little while ago when Jeremy first discovered it.  But we're slowly pushing it is slowly spreading.  At this point, we should probably make it a part of our best practice and update the OEP.  I'll put in a ticket on the BOM backlog to do that.